src/util/Constants.js

```diff
@@ -73,6 +73,8 @@ exports.Endpoints = {
         makeImageUrl(`${root}/team-icons/${teamId}/${hash}`, { size, format }),
       Sticker: (stickerId, stickerFormat) =>
         `${root}/stickers/${stickerId}.${stickerFormat === 'LOTTIE' ? 'json' : 'png'}`,
+      RoleIcon: (roleId, hash, format = 'webp', size) =>
```

Comment:
Probably out of the scope of this PR but why does every method here default the format to webp? Why not just do that on the makeImageUrl function?

Reply:
`makeImageUrl` does default `format` to `webp`. Since, some of the methods there don't call `makeImageUrl`, they have to default it themselves, which seems to be done even for other methods. There is one more inconsistency, the methods are taking format and size sometimes together in an object and sometimes not.